test/clj/chronograph/handlers/organization_test.clj

```diff
@@ -0,0 +1,75 @@
+(ns chronograph.handlers.organization-test
+  (:require [chronograph.domain.acl :as acl]
+            [chronograph.factories :as factories]
+            [chronograph.fixtures :as fixtures]
+            [chronograph.handlers.organization :as organization]
+            [clojure.spec.alpha :as s]
+            [clojure.test :refer :all])
+  (:import org.postgresql.util.PSQLException))
+
+(use-fixtures :once fixtures/config fixtures/datasource)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest create-new-organization-first-time
+  (testing "Creating a new organization returns a valid organization map in the response, and the creator is registered as admin in the ACL."
+    (let [user (factories/create-user)
+          response (organization/create {:body {:name "foo" :slug "bar"}
+                                         :user user})]
+      (is (= 200 (:status response)))
+      (is (s/valid? :organizations/organization
+                    (:body response)))
+      (is (acl/admin? (:users/id user)
+                      (:organizations/id (:body response)))))))
+
+
+(deftest create-organization-when-name-exists
```

Comment:
I think this should go into the domain tests. Let's try to keep the handler tests specific to the handler logic (validation, returning the appropriate HTTP status, response format, etc).

Reply:
Done; actually, deleted the test because turns out that its redundant. Creating an organization with the different slugs is as if we are creating two different organizations anyway. There exists a test for successful create organization.